### ISSUE TITLE
Training doesn't start and GPU access problem

### ISSUE THREAD

**Issue by SID-SURANGE** [fizyr/keras-retinanet#1325]
Hi,

I am training using a custom Dataset with around 280 images with Resnet50 as backbone. I tried running the code on COLAB where it ran fine but then when I switched to local GPU - TItan xp the GPU usage isn't there and training was even slow.

At present I am using Keras 2.3.1 and TF-gpu 2.1.0 but the training doesn't start(It doesn't move further after display of EPOCH1 text). I tried below things 
1) Reduce the steps/epochs and include --no-evaluation
2)  Changing tf-gpu to lower version but that gave error with CuDnn libraries

So  I am confused around - 
1) Why training doesn't start even gpu has been loaded
2) Are there any specific keras, TF versions that I need to use.
3) Is there any simple error that I am missing.

**Below is training parameters -**
`!./keras_retinanet/bin/train.py --freeze-backbone --random-transform --batch-size 4 --gpu=0 --steps 100 --epochs 20 --image-min-side=300 --image-max-side=300 --lr 0.001 --no-evaluation  csv annotations.csv classes.csv`

Another training tried but no luck -
`#!./keras_retinanet/bin/train.py --freeze-backbone --random-transform --batch-size 4 --gpu=0 --steps 400 --epochs 20 --image-min-side=300 --image-max-side=300 --lr 0.001 --weighted-average --compute-val-loss  csv annotations.csv classes.csv --val-annotations val_annotations.csv`

**GPU accessed -** 
```
2020-04-09 09:19:00.255331: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x559816261e70 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2020-04-09 09:19:00.255331: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): TITAN Xp, Compute Capability 6.1
1 Physical GPUs, 1 Logical GPUs
```

![image](https://user-images.githubusercontent.com/44268077/78856995-d306ed00-7a45-11ea-8afa-136b8a1bb57a.png)

Let me know if any further info required. Thanks in Advance.

**Comment by proxip** [comment]
try to reduce your batch size, I got same problem, and reduce my batch size and it's working

**Comment by SID-SURANGE** [comment]
> try to reduce your batch size, I got same problem, and reduce my batch size and it's working

I think BS of 4 is low, how reducing further helps this training. Can you give insights?

**Comment by LudovicLupus** [comment]
Hi @SID-SURANGE , did you ever figure this out? I'm getting an error pertaining to CuDNN library version mismatches, but I'm unsure how to resolve. The error occurs right after the Epoch 1/10 line.
